test/framework/src/main/java/org/opensearch/test/InternalTestCluster.java

```diff
@@ -2279,6 +2279,13 @@ public String startNode() {
         return startNode(Settings.EMPTY);
     }
 
+    /**
+     * Starts a search node with default settings and returns its name.
+     */
+    public String startSearchNode() {
```

Comment:
Do we need a new method here? I think `startSearchOnlyNodes(1)` will do the same thing.

Reply:
ah, it's just a shorthand and I copied it from `startDataOnlyNode`, I can replace it with `startSearchOnlyNodes(1)`